src/tests/ut/warehouse/warehouse_test.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "clinic_facade.hpp"
+#include "medicine.hpp"
+#include "product.hpp"
+#include "test_utils.hpp"
+#include "warehouse.hpp"
+#include "gtest/gtest.h"
+
+struct WarehouseTest : ::testing::Test
+{
+    std::shared_ptr<Warehouse> testWarehouse;
+    std::shared_ptr<Product> anaesthetic;
+    std::shared_ptr<Product> painkiller;
+
+  protected:
+    void SetUp()
+    {
+        Warehouse::createWarehouse();
+        testWarehouse = Clinic::getWarehouse();
+        anaesthetic = std::make_shared<Medicine>(std::string{"anaesthetic"}, 45, 100, 10122025, std::pair{-10, 0},
+                                                 std::vector<std::string>{{"chemicals"}});
+        painkiller = std::make_shared<Medicine>(std::string{"painkiller"}, 20, 100, 05062026, std::pair{0, 20},
```

Comment:
Please name somehow magic numbers such as 45,100, std::pair..
I do not know what they mean

Reply:
Fixed